.bazelversion

```diff
@@ -1 +1 @@
-3.6.0
+3.7.0
```

Comment:
?

Reply:
(Sorry didn't mean to approve, I think this needs to be reverted.)

/wait

Reply:
done; didn't mean to commit that. Having some trouble with fuzz-coverage tests anyway; not sure what the root cause is. The log didn't help me but I'll ask @asraa for help tomorrow.

Reply:
fuzz_coverage passed this time; must be an infrastructural flake.